src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -2543,15 +2522,14 @@ def test_mkern_invoke_multiple_any_spaces(tmpdir):
             in gen)
 
 
-@pytest.mark.xfail(reason="bug : loop fuse replicates maps in loops")
 def test_loopfuse():
     ''' Tests whether loop fuse actually fuses and whether
     multiple maps are produced or not. Multiple maps are not an
     error but it would be nicer if there were only one '''
     _, invoke_info = parse(os.path.join(BASE_PATH,
                                         "4_multikernel_invokes.f90"),
                            api=TEST_API)
-    psy = PSyFactory(TEST_API, distributed_memory=True).create(invoke_info)
+    psy = PSyFactory(TEST_API, distributed_memory=False).create(invoke_info)
```

Comment:
It would work with either, I just used this one as that was what the test was set up to use. I've parameterised it so it does both and also added in compilation of the resultant code.